cmd/kubectl-gadget/fsslower.go

```diff
@@ -41,7 +41,7 @@ var fsslowerCmd = &cobra.Command{
 	Short: "Trace open, read, write and fsync operations slower than a threshold",
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 		if fsslowerFilesystem == "" {
-			return utils.WrapInErrMissingArgs("--filesystem")
+			return utils.WrapInErrMissingArgs("--type")
```

Comment:
IMO, we should also change the Trace CR parameter. Otherwise, users using the kubectl-gadget CLI and trace will have different name parameters. WDYT?

Reply:
I changed it but I'm not sure `--type` is the best name here, however I want to keep it exactly the same as BCC fsslower https://github.com/iovisor/bcc/blob/5f2a64ec101b2916558bb934c436d4a6d4ed157c/libbpf-tools/fsslower.c#L107.

Reply:
Yeah, `type` seems too generic 🙁.

Reply:
Proposal: `fs-type`

Reply:
@mauriciovasquezbernal what is your plan for this PR?